src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -86,6 +85,15 @@ public class ExtractCohortEngine {
 
     private final Consumer<VariantContext> variantContextConsumer;
 
+    private static class VariantIterables {
+        public Iterable<GenericRecord> vets;
```

Comment:
Maybe we should brainstorm on a better name for the vets table and related? It now collides with the new name for VQSR Lite

Reply:
ugh yes, good point